### ISSUE TITLE
Charged ZnO defects - Defect minimizations

### ISSUE THREAD

**Issue by VondrakMar** [ddmms/ml-peg#307]
### Summary

This benchmark will be composed of different ZnO wurtzite structures containing an oxygen vacancy. The test set will include two charge states (0 and 2+), each exhibiting a distinct relaxed defect configuration. Multiple cell sizes should also be considered, as the defect can spuriously collapse in larger cells.

### Interactive features

A default test criterion would be the volume of the tetrahedral defect. The neutral defect is compressed compared with the tetrahedron containing oxygen, whereas the positive defect is expanded.

Additionally, one may plot the charge localized at the defect, if possible.

The defect site can also be visualized.

### Category

Physicality

### Data availability

The training data for MLIP, and publications corresponding to this test will be published in a following month. I have DFT optimized small cells, and we assume, that the similar defect should be preserved also for bigger systems. 

### Computational cost

On GPUs this should be relatively fast (below 10 minutes)
